keyboards/fortitude60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,165 @@
+#include "fortitude60.h"
```

Comment:
please replace `#include "fortitude60.h` with `#include QMK_KEYBOARD_H`. This is the standard for all keymap.c files these days.

Reply:
OK!